resinhup.py

```diff
@@ -117,6 +117,13 @@ def main():
         log.error("Resinhup is not supported for the resinOS version requested.")
         return False
 
+    # Maximum resinOS version
+    # XXX not supported yet
```

Comment:
It's a common marker for "problematic or misguiding code": https://en.wikipedia.org/wiki/Comment_(computer_programming)
I don't think it's quite the same as a FIXME.